config/settings.yml

```diff
@@ -0,0 +1,6 @@
+links:
+  rails_tutorial:
+    help_page: http://www.railstutorial.org/help
+    book_page: https://www.railstutorial.org/book
+    home_page: http://www.railstutorial.org/
+    
```

Comment:
add endline please

Reply:
sorry, what is meant by add endline

Reply:
It's mean you need add an empty line at end of file

Reply:
but i did leave an empty line already...
can you please help me to check my latest one thank you?